fiorella/src/styles/abstracts/_variables.scss

```diff
@@ -0,0 +1,12 @@
+
+$mobile: "only screen and (min-width: 767px)";
+$tablet: "only screen and (min-width: 768px) and (max-width: 1023px)";
+$desktop: "only screen and (min-width: 1024px)";
+
+$cerise-red-hover: rgba(#e83671, 0.7);
+$cerise-red: #e83671;
+$font-color-white: #ffffff;
+$border-color-white: #ffffff;
+$transparent: transparent;
+$playfair-display: "Playfair Display";
```

Comment:
You should add a default font here like `sans-serif` to be displayed if the browser doesn't found "Playfair Display" for any reason.